README.md

```diff
@@ -302,7 +302,10 @@ module.exports = {
 
 ### `sassOptions`
 
-Type: `Object|Function`
+Type: 
+```ts
+type sassOptions = object | (content: string|Buffer, loaderContext: LoaderContext, meta: any) => object 
```

Comment:
I think prettier should insert `;` at the end and add space between `string|Buffer`, i.e. `string | Buffer`, can you try to run prettier on readme?

Reply:
Ran prettier. Also, we should start running prettier with --write in the pre-commit hook?.

Reply:
That we already have, https://github.com/webpack-contrib/sass-loader/blob/babe42a1144e201cb17e3b076a677b167a7c2d41/lint-staged.config.js#L2

Reply:
> That we already have,

Not sure why it didn't trigger when I committed via terminal :(

Reply:
Triggering for me, did you install dependencies?

Reply:
Yea 🤔